tests/test_spelling.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""Testing the spelling of numbers
```

Comment:
@blagasz you can rename with git directly `git mv` or rename on the filesystem, add the new file and commit with the `-a` flag so git recognizes the old file removal.